test/java/org/opendatakit/briefcase/util/BriefcaseVersionManagerTest.java

```diff
@@ -0,0 +1,60 @@
+package org.opendatakit.briefcase.util;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.opendatakit.briefcase.reused.http.RequestBuilder.get;
+import static org.opendatakit.briefcase.reused.http.RequestBuilder.url;
+import static org.opendatakit.briefcase.util.BriefcaseVersionManager.getLatestUrl;
+
+import org.junit.Test;
+import org.opendatakit.briefcase.reused.http.FakeHttp;
+import org.opendatakit.briefcase.reused.http.response.ResponseHelpers;
+
+public class BriefcaseVersionManagerTest {
+  @Test
+  public void knows_the_url_to_get_the_latest_version() {
+    assertThat(getLatestUrl(), is(url("https://github.com/opendatakit/briefcase/releases/latest")));
+  }
+
+  @Test
+  public void knows_to_get_the_latest_available_version() {
+    FakeHttp http = new FakeHttp();
+
+    http.stub(
+        get(url("https://api.github.com/repos/opendatakit/briefcase/releases/latest")).build(),
```

Comment:
This string appears 4 times in the source. Maybe it could be a constant in `BriefcaseVersionManager`?